README.md

```diff
@@ -3,7 +3,7 @@
 [![Apache 2.0 License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.logevents/logevents/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.logevents/logevents)
 [![Javadoc](https://img.shields.io/badge/javadoc-logevents-blue)](https://jhannes.github.io/logevents/apidocs/)
-[![Build Status](https://github.com/jhannes/logevents/workflows/Java%20CI/badge.svg)](https://github.com/jhannes/logevents/actions/workflows/maven.yml)
```

Comment:
@jhannes I fucked up with a missing `]`. 

![image](https://user-images.githubusercontent.com/272215/218987090-1d94ea72-4f6e-4cf8-aaab-08ec35e3c6bd.png)

Reply:
@jhannes Since I broke it, here is a follow up pull request: https://github.com/jhannes/logevents/pull/81